pkg/server/api/sharkey.go

```diff
@@ -86,6 +86,12 @@ func Run(conf *config.Config, logger *logrus.Logger) {
 		Handler:   loggingHandler,
 	}
 
+	if c.conf.GitHub.Enabled {
```

Comment:
Some thoughts here:

1. StartGitHubUserMappingSyncJob may block for a bit on initial mapping update. It could be started in the background, but maybe it's not a big deal as in the worst case it will time-out.
1. If there's a problem initializing the cron job, and Sharkey is configured to use GitHub SSH CA integration, then the user mappings won't be updated. Apart from one error in the logs, it won't be clear what's going on.
1. The background job will run on every server that Sharkey is deployed to. This has couple implications, on each start-up Sharkey will call GitHub API, if there's a crash loop it will continue calling GH and probably will hit rate limits. Second, each server might be hitting GH at roughly the same time and each will create a batch SQL query to update data. Hopefully this is not a massive problem, but would be good to either have a way to only run the job on some hosts or add some random jitter.